Screenbar/MainWindowView.xib

```diff
@@ -12,18 +12,20 @@
                 <outlet property="errorMessage" destination="KWl-fd-KYO" id="yw4-AI-c8O"/>
                 <outlet property="path" destination="MIa-ei-b7S" id="cHf-jh-Uie"/>
                 <outlet property="playSound" destination="yQo-Gb-vL8" id="Jz6-eF-i3q"/>
+                <outlet property="quit" destination="zfO-Iw-jKV" id="xnl-cT-naP"/>
```

Comment:
Position this button to the left so that is aligned with the rest of the controls above (The checkbox for sound etc.)